src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,62 @@
  */
 
 public class Plane {
+    private String type;
+    private String typeEngine;
+    private int range;
+    private int maxSpeed;
+    private int numberOfEngines;
+    private int capacity;
 
+    public Plane(PlaneBuilder builder) {
```

Comment:
```suggestion
    private Plane(PlaneBuilder builder) {
```
explain why in the comment below

Reply:
because no access is needed outside this class